plancton/__init__.py

```diff
@@ -210,18 +211,19 @@ def _read_conf(self):
       self.conf["max_docks"] = 0
     if not isinstance(self.conf["docker_cmd"], list):
       self.conf["docker_cmd"] = self.conf["docker_cmd"].split(" ")
-    if self.conf["influxdb_url"] and not "#" in self.conf["influxdb_url"]:
-      # Invalid InfluxDB URL: disable monitoring
-      self.conf["influxdb_url"] = None
+    tmp_url_list = []
```

Comment:
By the way you can make `self.conf["influxdb_url"]` a set here, instead of temporarily converting it to a set when checking for changes.